judge/admin/interface.py

```diff
@@ -79,11 +79,27 @@ class BlogPostAdmin(VersionAdmin):
     form = BlogPostForm
     date_hierarchy = 'publish_on'
 
+    def has_view_permission(self, request, obj=None):
```

Comment:
Is this necessary? None of the other classes override `has_view_permission`.

Reply:
Might as well document this: after a long search, it seems that `get_queryset` is filtered to fetch the model instance, so as long as we write that method write, we don't have to write `has_view_permission`.